protocol/src/main/contraband-scala/sbt/protocol/codec/ExectionEventFormats.scala

```diff
@@ -5,21 +5,21 @@
 // DO NOT EDIT MANUALLY
 package sbt.protocol.codec
 import _root_.sjsonnew.{ deserializationError, serializationError, Builder, JsonFormat, Unbuilder }
-trait ExectionEventFormats { self: sjsonnew.BasicJsonProtocol =>
-implicit lazy val ExectionEventFormat: JsonFormat[sbt.protocol.ExectionEvent] = new JsonFormat[sbt.protocol.ExectionEvent] {
-  override def read[J](jsOpt: Option[J], unbuilder: Unbuilder[J]): sbt.protocol.ExectionEvent = {
+trait ExecutionEventFormats { self: sjsonnew.BasicJsonProtocol =>
```

Comment:
This is an API break afaict.

And yes, I can see there's a DO NOT EDIT MANUALLY marker.
I didn't edit it manually, a script did ;-)

1. If you're intentionally misspelling things, you should include a note explaining why.
1. If you're generating things, you should indicate the source, not just the tool.